holoviews/plotting/bokeh/widgets.py

```diff
@@ -1,8 +1,20 @@
 import json
+from distutils.version import LooseVersion
+
 import param
+import bokeh
+from bokeh.io import push_notebook, Document
+
+if LooseVersion(bokeh.__version__) >= LooseVersion('0.11'):
+    old_bokeh = False
+    from bokeh.io import _CommsHandle
+    from bokeh.util.notebook import get_comms
```

Comment:
How much can we trust that ``_CommsHandle`` is something we can rely on in future? From the name, it doesn't look like something we are expected to use directly...